ui/src/dialogs/AuthenticationDialog/AuthenticationDialog.scss

```diff
@@ -1,12 +1,8 @@
 @import 'app/mixins.scss';
 
-.AuthenticationScreen {
+.AuthenticationScreen.AuthenticationScreen {
   width: 28em !important;
```

Comment:
When I was a kid, !important was not considered best practice. Has this changed?